homeassistant/components/renson/binary_sensor.py

```diff
@@ -0,0 +1,133 @@
+"""Binary sensors for renson."""
+from dataclasses import dataclass
+
+from renson_endura_delta.field_enum import (
+    AIR_QUALITY_CONTROL_FIELD,
+    BREEZE_ENABLE_FIELD,
+    BREEZE_MET_FIELD,
+    CO2_CONTROL_FIELD,
+    FROST_PROTECTION_FIELD,
+    HUMIDITY_CONTROL_FIELD,
+    PREHEATER_FIELD,
+    DataType,
+    FieldEnum,
+)
+from renson_endura_delta.renson import RensonVentilation
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RensonCoordinator
+from .const import DOMAIN
+from .entity import RensonEntity
+
+
+@dataclass
+class RensonBinarySensorEntityDescriptionMixin:
+    """Mixin for required keys."""
+
+    field: FieldEnum
+
+
+@dataclass
+class RensonBinarySensorEntityDescription(
+    BinarySensorEntityDescription, RensonBinarySensorEntityDescriptionMixin
+):
+    """Description of binary sensor."""
+
+
+BINARY_SENSORS: tuple[RensonBinarySensorEntityDescription, ...] = (
+    RensonBinarySensorEntityDescription(
+        name="Frost protection active",
```

Comment:
Instead of the name here in English, add a translation_key and define the English names in renson/strings.json: https://developers.home-assistant.io/docs/internationalization/core#entities

To test the translations, run `python3 -m script.translations develop --all`